c3/utils/qt_utils.py

```diff
@@ -245,141 +209,127 @@ def pad_matrix(matrix, dim, padding):
     return matrix
 
 
-def perfect_gate(  # noqa
-    gates_str: str, index=[0, 1], dims=[2, 2], proj: str = "wzeros"
-):
+# def perfect_gate(  # noqa
```

Comment:
Since we are doing a cleanup of the `qt_utils`, I would recommend removing all commented code that doesn't have a `# TODO` to go along with it. We can always find old code in git commit logs and having commented code strewn in various places over the source code would typically confused new developers. If there is something to be conveyed by keeping the commented section, eg, some insight or warning for future developers, it would be sensible to delete the commented code and add a comment of the form:
```python
# NOTE: Removed perfect_gate() as in <commit-hash> because <reason why removed>
```

`qt_utils` has several commented sections later in the file as well and it would be useful to clean all of that up and/or add helpful `# TODO`s wherever necessary.

Reply:
I removed the ones I know but there remain some options that Fed might need. @fedroy can you take a look and check what to do with the Clifford decompositions?

Reply:
I made some adjustments to make the example notebook work. The Cliffords are now in `librarires/constants.py`